schema/datatypes/datatypes.fbs

```diff
@@ -96,23 +96,24 @@ enum BodyPart: uint8 {
     NONE = 0,
     HEAD = 1,
     NECK = 2,
-    CHEST = 3,
-    WAIST = 4,
-    HIP = 5,
-    LEFT_UPPER_LEG = 6,
-    RIGHT_UPPER_LEG = 7,
-    LEFT_LOWER_LEG = 8,
-    RIGHT_LOWER_LEG = 9,
-    LEFT_FOOT = 10,
-    RIGHT_FOOT = 11,
-    LEFT_LOWER_ARM = 14,
-    RIGHT_LOWER_ARM = 15,
-    LEFT_UPPER_ARM = 16,
-    RIGHT_UPPER_ARM = 17,
-    LEFT_HAND = 18,
-    RIGHT_HAND = 19,
-    LEFT_SHOULDER = 20,
-    RIGHT_SHOULDER = 21,
+    UPPER_CHEST = 3,
+    CHEST = 4,
+    WAIST = 5,
+    HIP = 6,
+    LEFT_UPPER_LEG = 7,
+    RIGHT_UPPER_LEG = 8,
+    LEFT_LOWER_LEG = 9,
+    RIGHT_LOWER_LEG = 10,
+    LEFT_FOOT = 11,
+    RIGHT_FOOT = 12,
+    LEFT_LOWER_ARM = 13,
+    RIGHT_LOWER_ARM = 14,
+    LEFT_UPPER_ARM = 15,
+    RIGHT_UPPER_ARM = 16,
+    LEFT_HAND = 17,
+    RIGHT_HAND = 18,
+    LEFT_SHOULDER = 19,
+    RIGHT_SHOULDER = 20,
```

Comment:
you're not supposed to change the IDs of the enum members.

Reply:
I would prefer to and it’s not 1.0